contrib/nydusify/cmd/nydusify.go

```diff
@@ -364,7 +364,12 @@ func main() {
 					Usage:   "Read prefetch list from STDIN, please input absolute paths line by line",
 					EnvVars: []string{"PREFETCH_PATTERNS"},
 				},
-
+				&cli.StringFlag{
+					Name:    "compressor",
+					Value:   "lz4_block",
+					Usage:   "Set algorithm to compress chunks, possible values: lz4_block, zstd",
```

Comment:
Maybe `Algorithm to compress image data blob, possible values: none, lz4_block, zstd`.

Reply:
fixed